docs/local_resource.md

```diff
@@ -117,10 +117,20 @@ With `serve_cmd`, when the resource updates:
 Some examples:
 
 #### build and run a server locally
-``local_resource(cmd='go build ./cmd/myserver', serve_cmd='./myserver --port=8001', deps=['cmd/myserver'])``
+``local_resource('local-server', cmd='go build ./cmd/myserver', serve_cmd='./myserver --port=8001', deps=['cmd/myserver'])``
 
 #### keep a port forward open to a service not deployed by Tilt
-``local_resource(serve_cmd='kubectl port-forward -n openfaas svc/gateway 8080:8080')``
+``local_resource('kube-port-forward', serve_cmd='kubectl port-forward -n openfaas svc/gateway 8080:8080')``
 
 #### show the k8s api server's logs
-``local_resource(serve_cmd='kubectl logs -f -n kube-system kube-apiserver-docker-desktop')``
+``local_resource('kube-logs', serve_cmd='kubectl logs -f -n kube-system kube-apiserver-docker-desktop')``
+
+#### custom environment
+```python
+update_env={'CGO_ENABLED': '0'}
```

Comment:
not necessary, but if you want to be extra clear, consider inline comments:
```
update_env={'CGO_ENABLED': '0'}  # will override any values for this key currently in the env
serve_env={'PORT': os.getenv('PORT', default='8001')}  # if $PORT set, use that value, else use '8001'
```

Reply:
Good call - I put them on their own lines just to prevent horizontal scroll at most browser widths (we get about 65 cols max or so before it'll force a scroll)

<img width="767" alt="Screen Shot 2021-01-25 at 9 42 51 AM" src="https://user-images.githubusercontent.com/841263/105720689-b616aa00-5ef1-11eb-907f-97b1dfd81d81.png">